cloudsync/providers/file.py

```diff
@@ -0,0 +1,504 @@
+import os
```

Comment:
can you rename the file from file.py to filesystem.py?

Reply:
you know you are the one who renamed it in the first place

Reply:
I didn't rename the file, I renamed the class